test/java/org/opendatakit/briefcase/reused/UncheckedFilesTest.java

```diff
@@ -0,0 +1,65 @@
+package org.opendatakit.briefcase.reused;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createFile;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.createTempDirectory;
+import static org.opendatakit.briefcase.reused.UncheckedFiles.deleteRecursive;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class UncheckedFilesTest {
+  private static Path tempDir;
+  private Path temp;
+
+
+  @Before
+  public void setUp() {
+    tempDir = createTempDirectory("briefcase");
+    temp = Paths.get(tempDir.toString() + "/test.txt");
+    createFile(temp);
+  }
+
+  @After
+  public void tearDown() {
+    deleteRecursive(tempDir);
+  }
+
+  @Test(expected = UncheckedIOException.class)
```

Comment:
I think this test is not really necessary because then it feels like we should test all the other possible causes for `Files.newInputStream` to throw an `IOException`